Image_Viewer/theme.py

```diff
@@ -0,0 +1,35 @@
+import json
+import os
+class Theme:
+    def __init__(self):
+        themeDictionary = self.read_theme()
+        self.themes = themeDictionary["themes"]
+        self.current_theme=themeDictionary["lastTheme"]
+        
+    
+        
+    
+    def set_theme(self, background, foreground, buttonBackground):
+            self.root.configure(bg=background)
+            self.name_label.configure(bg=background, fg=foreground)
+            self.display.configure(bg=background)
+            self.move_left_button.configure(bg=buttonBackground, fg=foreground)
+            self.move_right_button.configure(bg=buttonBackground, fg=foreground)
+            
+
+    def read_theme(self):
+        with open('Image_Viewer/themes.json', 'r') as openfile:
+            json_object = json.load(openfile)
+            return json_object
+    
+    def write_theme(self):
+        themeDictionary = {"themes":self.themes,"lastTheme":self.current_theme}
```

Comment:
I think we can improve this writing bit further - the only thing that changes is what the user's last theme was.

I think it would be a good idea to:
1) Keep all the themes in themes.json, I really like having the themes out of the code. It'll make it super easy to add a new theme.
2) Have "lastTheme" in a separate JSON file, maybe something like customsettings.json to open up possibilities for further saved settings in the future. That customsettings.json file could be added to the gitignore (which is currently just on latest master), so it wouldn't be picked up in Git.

Arguably, if there are more settings in the future, that could mean we run into the same thing around changing a whole file when only one thing is updated, but I think the difference here is that the themes themselves are constant, whereas user preferences aren't, so I think it's better to separate them out.

How does this sound?

Reply:
Sorry, missed this part due to a series of comment, will implement this.